libs/payments/stripe/src/lib/stripe.client.ts

```diff
@@ -10,12 +10,15 @@ import { StripeClientConfig } from './stripe.client.config';
 @Injectable()
 export class StripeClient {
   public readonly stripe: Stripe;
+  public taxIds: { [key: string]: string };
```

Comment:
Imo taxIds should move to the manager, since it's unrelated to reaching out to stripe.

Additionally, we should make members like this private.